test/CppTests/debuggees/natvis/src/main.cpp

```diff
@@ -3,6 +3,7 @@
 #include "SimpleVector.h"
 #include "SimpleArray.h"
 #include "SimpleClass.h"
+#include "C.h"
```

Comment:
nit: Can we keep the naming of the test classes consistent with the others?

E.g. `SimpleMatrix`